core/vm/vm.go

```diff
@@ -28,27 +28,49 @@ import (
 	"github.com/ethereum/go-ethereum/params"
 )
 
-// Vm is an EVM and implements VirtualMachine
-type Vm struct {
-	env Environment
+// Config are the configuration options for the EVM
+type Config struct {
+	Debug     bool
+	EnableJit bool
+	ForceJit  bool
+	Logger    LogCfg
+}
+
+// LogCfg are the configuration options for structured logger the EVM
+type LogCfg struct {
+	DisableMemory  bool // disable memory capture
+	DisableStack   bool // disable stack capture
+	DisableStorage bool // disable storage capture
+	FullStorage    bool // show full storage (slow)
 }
```

Comment:
Can the log options be in Config? They'd need slightly different names maybe, but I don't see why they're separate.